projects/plugins/jetpack/modules/likes.php

```diff
@@ -24,12 +26,15 @@
 	'dns-prefetch'
 );
 
-include_once dirname( __FILE__ ) . '/likes/jetpack-likes-master-iframe.php';
-include_once dirname( __FILE__ ) . '/likes/jetpack-likes-settings.php';
+require_once dirname( __FILE__ ) . '/likes/jetpack-likes-master-iframe.php';
+require_once dirname( __FILE__ ) . '/likes/jetpack-likes-settings.php';
 
+/** Jetpack Like Class */
 class Jetpack_Likes {
+
+	/** Initialize class */
```

Comment:
While this is accepted by phpcs, personally I'd format the doc blocks consistently in long format to match those that already exist in the file.
```suggestion
	/**
	 * Initialize class
	 */
```